react/Footer/Footer.js

```diff
@@ -50,7 +50,7 @@ export default class Footer extends Component {
     const { locale, linkRenderer, authenticationStatus } = this.props;
 
     return (!specificLocale || specificLocale === locale) ?
-      <FooterLink secondary={secondary} linkRenderer={linkRenderer} authenticationStatus={authenticationStatus} key={i} {...restProps}>{name}</FooterLink> :
+      <div key={i}><FooterLink secondary={secondary} linkRenderer={linkRenderer} authenticationStatus={authenticationStatus} {...restProps}>{name}</FooterLink></div> :
```

Comment:
Can use Fragment here

Reply:
You are correct @kosanna , made obsolete with my latest changes. Removed!